src/System.Runtime.InteropServices.RuntimeInformation/src/System.Runtime.InteropServices.RuntimeInformation.csproj

```diff
@@ -4,11 +4,8 @@
     <RootNamespace>System.Runtime.InteropServices</RootNamespace>
     <AssemblyName>System.Runtime.InteropServices.RuntimeInformation</AssemblyName>
     <ProjectGuid>{1CBC030D-B5D3-4AB5-A9FD-24EC5F6F38D2}</ProjectGuid>
-    <DefineConstants Condition="'$(TargetGroup)'=='net45' OR '$(TargetGroup)'=='netfx' OR '$(TargetGroup)'=='net462' OR '$(TargetGroup)'=='net47'">$(DefineConstants);netfx</DefineConstants>
```

Comment:
I would add this one back with the "TargetsNetFx" = true condition.

Reply:
Should not be necessary since "netfx" define is already changed to use that condition.